packages/sources/northwestnodes/src/config/index.ts

```diff
@@ -0,0 +1,24 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const defaultEndpoint = 'staking_ethereum_epoch_single'
+
+export const config = new AdapterConfig({
+  API_KEY: {
+    description: 'The Northwest Nodes API key',
+    type: 'string',
+    required: true,
+    sensitive: true,
+  },
+  API_ENDPOINT: {
+    description: 'The default HTTP API base url',
+    type: 'string',
+    required: false,
+    default: 'https://api.northwestnodes.com/v2',
+  },
+  WARMUP_SUBSCRIPTION_TTL: {
+    description: 'Reduced warmup subscription TTL to 10 seconds, for this EA',
+    type: 'number',
+    required: false,
+    default: 10000,
+  },

```

Comment:
This will actually fail as startup, because adding it here means you add it as a custom setting and you're not allowed to override a base framework setting. To override a default you can pass a second parameter to the `AdapterConfig` with an `envDefaultOverrides` prop

Reply:
Removed entirely based on conversations.